### ISSUE TITLE
Texify down?

### ISSUE THREAD

**Issue by pseeth** [agurodriguez/github-texify#11]
Hi, thanks for a great library! I'm wondering, is Texify down or not working currently? The last commit from texify on one of my repositories was yesterday and there haven't been any commits since, despite updates.

Thanks!

**Comment by agurod42** [comment]
Hi @pseeth.

I'm glad you find it useful.

I don't see any failed delivery here: 

<img width="1299" alt="Screen Shot 2019-04-10 at 7 19 44 PM" src="https://user-images.githubusercontent.com/13837769/55919898-a10a4300-5bc5-11e9-8e35-4cf85c4eb4f8.png">

Could you point me to the repo you're working on?

**Comment by pseeth** [comment]
The repo I'm working on is private, but Texify should have access to all the repositories my organization has access to, private or public, right?

**Comment by pseeth** [comment]
Texify appears to work for my public repositories now after uninstalling and reinstalling the Github app from my organization but it's broken for my private repositories.

**Comment by SebastianM-C** [comment]
Hello! Thanks for making this app. I came here with the same question as OP. I [renamed README.md to README.tex.md](https://github.com/SebastianM-C/Nuclear-surface-vibrations/commit/eb3672f37bc12dcc762f0a5531588e6eebb14d65) and also [changed the file in a separate commit](https://github.com/SebastianM-C/Nuclear-surface-vibrations/commit/fad83e0ac3b54182a4e9fe4e2f02a1abcfec486b), but I didn't see any activity from the app. Am I missing something?

**Comment by JeffDeCola** [comment]
Hi, this app is great!!

I'm also having a problem with it not triggering on my pushes.  It was working great, but then it just stopped.  I tried to delete and make a new README.text.md, uninstalled/reinstalled texify but no more activity.

**Comment by ghost** [comment]
@agurodriguez First of all, this app is great.

But it only works normally for public repository, not so good for private repositories.
This is how we can trigger the problem.
1. Create a new filename.tex.md file and add some latex into the file
2. Commit it and texify will render it properly
3. Change the file name to filename_filename.tex.md, add some new latex equation into the file
4. Texify can't render the latex this time. 
5. Make the private repository public, repeat the process, texify starts working.

Please take some time to fix the problem, Texify would be much better if this problem get solved.

**Comment by DannyDannyDanny** [comment]
I have Texify enabled on [this public repo](https://github.com/DannyDannyDanny/reuters-21578-json/) but even after adding a `*.tex.md` file, nothing changes. Here's the [uploaded file](https://github.com/DannyDannyDanny/reuters-21578-json/blob/master/maths.tex.md).

**Comment by pseeth** [comment]
Yeah Texify has stopped working for me entirely on both public and private repositories.
